samples/sample_presets.py

```diff
@@ -0,0 +1,35 @@
+"""Presets sample."""
+from zivid import Application
+from zivid.presets import categories
+
+
+def _main():
+    app = Application()
+    camera = app.connect_camera()
+
+    available_categories = categories(camera.info.model)
+    print("The following preset categories are available:")
+    for index, category in enumerate(available_categories):
+        print("    {}: {}".format(index, category.name))
+    chosen_category = available_categories[
+        int(input("Choose a category (enter a number): "))
+    ]
+
+    print(
+        "The following presets are available in category '{}':".format(
+            chosen_category.name
+        )
+    )
+    for index, preset in enumerate(chosen_category.presets):
+        print("    {}: {}".format(index, preset.name))
+    chosen_preset = chosen_category.presets[
+        int(input("Choose a preset (enter a number): "))
+    ]
+
+    print("Capturing point cloud with preset '{}' ...".format(chosen_preset.name))
+    with camera.capture(chosen_preset.settings) as frame:
+        frame.save("result.zdf")
+
```

Comment:
Love this sample. Maybe also add this at the end:

```
settings_file = chosen_preset.name + ".yml"
print(f"Saving Settings to {settings_file}")
chosen_preset.settings.save(settings_file)
```

Reply:
@FYI @SatjaSivcev, we are adding a presets sample for Python here, we may want to port this over to https://github.com/zivid/zivid-python-samples.
Can CSU make an issue for this?